sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/CollectionConfiguration.cs

```diff
@@ -121,6 +142,92 @@ private void UpdateMetricIdOfError(CollectionConfigurationError error, string id
             }
         }
 
+        private void CreateDocumentStreams(
+            out CollectionConfigurationError[] errors,
```

Comment:
nit: change the order of parameter. Normally, out variables are listed towards the end.

Reply:
Reordered the parameters.